Add section about grover tool (#42)
Ctrl-Alt-Defeat-Foundation-Models/informatiCup2024Paper: paper.tex

\textbf{Winston AI} is a commercial AI text detector that starts at 18\$ a month. They have two primary methods for checking if a text is AI written. Firstly linguistic analysis analyses every text for characteristics, that are different between human- and AI-generated work. One example characteristic is the repetitiveness, another is Data Training, where the detector saw many AI written texts of one model and learned unique characteristics that this model uses which then can be detected. So this method is more generator-specific. \cite{WinstonAI}

> Another tool to automatically detect generated text is \textbf{GLTR}. It applies different baseline statistical methods to detect generation artifacts. To do so, it needs access to a certain model, for which it can predict, whether the text is likely to be generated. The text is compared with the distribution of next words, in the given context. The tool calculates which word would have been predicted as next word at each token in the text and allows to colour which words are how likely to appear by the model at a given position and context. If a text is coloured completely green, a lot of words have a high chance, that they would have been predicted at this position by e.g. the currently hosted GPT2 117M model. Therefore, it is likely that a mostly green coloured text is generated by the GPT2 model. This approach can also be used for other models. \cite{gehrmann2019gltr}

Reviewer: ```suggestion
Another tool to automatically detect generated text is \textbf{GLTR}. It applies different baseline statistical methods to detect generation artefacts. To do so, it needs access to a certain model, for which it can predict, whether the text is likely to be generated. The text is compared with the distribution of possible consecutive words given the semantic context. The tool calculates which word would have been predicted as next word at each token in the text and allows to colour which words are how likely to appear by the model at a given position and context. If a text is coloured completely green, a lot of words have a high chance, that they would have been predicted at this position by e.g. the currently hosted GPT2 117M model. Therefore, it is likely that a mostly green coloured text is generated by the GPT2 model. This approach can also be used for other models. \cite{gehrmann2019gltr}
```

> A more recent approach is presented in \cite{zellers2020defending}. With the aim to investigate in the detection of AI generated fake news,  the authors created a model called \textbf{Grover}, which is on the hand able to generate fake news, but on the other hand also pretty reliably detects, whether a given text is generated by itself. The study also elaborates on how to deal with the issue on generated fake news. Therefore, inspired by IT security research, they advocate for the release of such models, to allow the development of effective countermeasures, like detector models.

Reviewer 2: ```suggestion
A more recent approach is presented in \cite{zellers2020defending}. With the aim to investigate in the detection of AI generated fake news,  the authors created a model called \textbf{Grover}, which is on the one hand able to generate fake news, but on the other hand also pretty reliably detects, whether a given text is AI-generated by itself. The study also elaborates on how to deal with the issue on generated fake news. Therefore, inspired by IT security research, they advocate for the release of such models, to allow the development of effective countermeasures, like detector models.
```

> A more recent approach is presented in \cite{zellers2020defending}. With the aim to investigate in the detection of AI generated fake news, the authors created a model called \textbf{Grover}, which is on the one hand able to generate fake news, but on the other hand also pretty reliably detects, whether a given text is AI-generated by itself. The study also elaborates on how to deal with the issue on generated fake news. Therefore, inspired by IT security research, they advocate for the release of such models, to allow the development of effective countermeasures, like detector models.

Reviewer: ```suggestion
A more recent approach is presented in \cite{zellers2020defending}. With the aim to investigate in the detection of AI generated fake news, the authors created a model called \textbf{Grover}, which is on the one hand able to generate fake news, but on the other hand also pretty reliably detects, whether a given text is AI-generated by itself. The study also elaborates on how to deal with the issue on generated fake news. Therefore, inspired by IT security research, they advocate for the release of such models to allow the development of effective countermeasures like detector models.
```

> A more recent approach is presented in \cite{zellers2020defending}. With the aim to investigate in the detection of AI generated fake news,  the authors created a model called \textbf{Grover}, which is on the one hand able to generate fake news, but on the other hand also pretty reliably detects, whether a given text is AI-generated by itself. The study also elaborates on how to deal with the issue on generated fake news. Therefore, inspired by IT security research, they advocate for the release of such models, to allow the development of effective countermeasures, like detector models.

Reviewer 3: ```suggestion
A more recent approach is presented in \cite{zellers2020defending}. With the aim to investigate in the detection of AI generated fake news, the authors created a model called \textbf{Grover}, which is on the one hand able to generate fake news, but on the other hand also pretty reliably detects, whether a given text is AI-generated by itself. The study also elaborates on how to deal with the issue on generated fake news. Therefore, inspired by IT security research, they advocate for the release of such models, to allow the development of effective countermeasures, like detector models.
```
